MonkeyLoader/Configuration/DefiningConfigKey.cs

```diff
@@ -36,6 +36,14 @@ public sealed class DefiningConfigKey<T> : Entity<DefiningConfigKey<T>>, IDefini
         private ConfigKeyChangedEventHandler? _untypedChanged;
         private T? _value;
 
+        /// <summary>
+        /// Implicit way to get the value for this config key
```

Comment:
Please make sure the documentation follows the standard scheme. Normally all parameters and such should be covered too, but that can be a bit funny for operators like this since it's hidden anyways.

In this case something like: _Implicitly <see cref="GetValue()">gets the value</see> of this config item._